WebRoot/js/zimbraAdmin/domains/model/ZaDomain.js

```diff
@@ -2692,6 +2699,7 @@ ZaDomain.myXModel = {
        {id:ZaDomain.A2_isTestingSync, ref:ZaDomain.A2_isTestingSync, type:_NUMBER_},
        {id:ZaDomain.A2_isTestingAuth, ref:ZaDomain.A2_isTestingAuth, type:_NUMBER_},
        {id:ZaDomain.A_zimbraFeatureCalendarReminderDeviceEmailEnabled, type:_ENUM_, choices:ZaModel.BOOLEAN_CHOICES, ref:"attrs/" + ZaDomain.A_zimbraFeatureCalendarReminderDeviceEmailEnabled},
+	{id:ZaDomain.A_zimbraFeatureAllowUsernameInPassword, type:_ENUM_, choices:ZaModel.BOOLEAN_CHOICES, ref:"attrs/" + ZaDomain.A_zimbraFeatureAllowUsernameInPassword},
```

Comment:
indentation

Reply:
in order to fix the indentation for the code that is part of this ticket we need to fix the indentation of the existing code indentation. So made the changes, and fixed indentation.